server/routes/TripRoute.js

```diff
@@ -19,8 +20,8 @@ class TripRoute {
 
   async getTripById(req, res) {
     try {
-      const response = await controllers.tripController.getTrip(req.userId)
-      res.status(200).json(response)
+      const response = await controllers.tripController.getTrip(req.params.id)
```

Comment:
Discussed in discord, still need to pass in userId incase trip is not public

Reply:
its filtered in the DB call